src/scripts/modules/media/body/content-common.less

```diff
@@ -345,6 +346,16 @@ table {
   background-color: @table-bg;
   counter-increment: table;
 
+  ul {
+    padding: 0;
+    margin: 0;
+    &[data-bullet-style="none"] {
+      li {
+        list-style: none;
+      }
+    }
+  }
+
```

Comment:
I am not sure why this is here but can we move it `content-baked.less` as well to avoid any regressions? @PiotrKozlowski @Adrianltd @PatBoj